packages/liveblocks-node/src/index.ts

```diff
@@ -11,12 +11,17 @@ type AuthorizeOptions = {
   room: string;
   /**
    * The id of the user that try to connect. It should be used to get information about the connected users in the room (name, avatar, etc).
+   * It can also be used to generate a token that gives access to a private room where the userId is configured in the room accessesl
```

Comment:
Typo?

```suggestion
   * It can also be used to generate a token that gives access to a private room where the userId is configured in the room accesses
```

Reply:
Fixed